src/LtiLibrary.NetCore/Clients/SecuredClient.cs

```diff
@@ -17,46 +17,43 @@ public static class SecuredClient
         /// <summary>
         /// Add a signed authorization header to the client request using the signatureMethod.
         /// </summary>
-        /// <param name="client">The HttpClient that will make the request.</param>
-        /// <param name="method">The HttpMethod of the request.</param>
-        /// <param name="serviceUrl">The service URL.</param>
-        /// <param name="content">The Content of the request (used to calculate the body hash).</param>
+        /// <param name="client">May need BaseAddress value from this</param>
+        /// <param name="request">The request object that will be sent</param>
         /// <param name="consumerKey">The OAuth consumer key.</param>
         /// <param name="consumerSecret">The OAuth consumer secret.</param>
         /// <param name="signatureMethod">The SignatureMethod used to sign the request.</param>
         /// <returns></returns>
-        public static async Task SignRequest(HttpClient client, HttpMethod method, string serviceUrl,
-            HttpContent content, string consumerKey, string consumerSecret, SignatureMethod signatureMethod)
+        public static async Task SignRequest(HttpClient client, HttpRequestMessage request, string consumerKey, string consumerSecret, SignatureMethod signatureMethod)
         {
-            if (client == null)
+            if (request == null)
```

Comment:
Probably should still keep:

```
if (client == null)
{
    throw new ArgumentNullException(nameof(client));
}

Reply:
Good call, I had removed the client for a bit, but brought it back to address edge cases with relative paths.  Restored check here: e630016